website/content/docs/api-gateway/usage/basic-usage.mdx

```diff
@@ -0,0 +1,135 @@
+---
+layout: docs
+page_title: Consul API Gateway Basic Usage
+description:  >-
+  Consul API Gateway Basic Usage
+---
+
+
+# Basic Usage
+
+This topic describes the basic workflow for implementing Consul API Gateway configurations.
+
+1. Verify that the [requirements](/docs/api-gateway/consul-api-gateway-install#requirments) have been met.
```

Comment:
i believe we deprecated this page. also, can we list the requirements here so users aren't clicking on a different page? this might reduce conversion rates

```suggestion
1. Verify that the [requirements](/docs/api-gateway/tech-specs) have been met.
```

Reply:
I had originally moved the requirements onto this page, but had been asked to move them back to the separate page.